### DISCUSSION TITLE
Peers in different LANs cannot connect to one another

### DISCUSSION THREAD

**Discussion by soq2000** [Novage/p2p-media-loader#199] [Q&A]
Dear all,
Finally I can build the project with a private tracker. Very nice GUI and feature. Thank you very much for such project
I have one question on the location of Peer. It seems to me that only Peers in the same LAN (created by clicking on the the Button "Create New Peer") can see and share the date. If I start the website from other computer (via other gateway to Internet) they cannot see each other. Or maybe I miss some special settings?
Many thanks
BR

**Comment by mrlika** [comment]
This technology is like BitTorrent - it doesn't guaranty that all the peers can connect with each other. It usually happens because of incorrect routing and NAT configuration of networks that makes not possible direct peers connections.
